### ISSUE TITLE
Feature suggestion: Left/right scroll when zoomed in

### ISSUE THREAD

**Issue by harryjubb** [calipho-sib/feature-viewer#20]
This would be a really great feature to have, being able to stay zoomed in and move along the sequence. From chatting with colleagues, this seems to be a logical feature for a sequence browser. Would this be tractable for Feature Viewer?

**Comment by sunworshipper** [comment]
I agree about the scroll feature.  It's an important functionality.  If you are zoomed in a region you have to zoom out before you zoom in again to another region.  Scroll left right would be great.

**Comment by MatSchaeff** [comment]
At neXtProt we are also facing a lag problem by displaying proteins with a lot of features, so that's a functionality that we will try to add soon ;)
Probably as Harry did for his viewer 
Harry already made a nice implementation of it : http://cancer.sanger.ac.uk/cosmic3d/protein/BRAF

**Comment by gabrielctn** [comment]
That is exactly the functionnality I am looking for !
Has it been implemented somewhere ? It would be really great to have it, the viewer would be almost perfect then :)
